[2025-01-28T09:35:12.372+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-28T09:35:12.386+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_data_ingestion.ingest_data manual__2025-01-28T09:35:10.291418+00:00 [queued]>
[2025-01-28T09:35:12.392+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_data_ingestion.ingest_data manual__2025-01-28T09:35:10.291418+00:00 [queued]>
[2025-01-28T09:35:12.393+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-28T09:35:12.402+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): ingest_data> on 2025-01-28 09:35:10.291418+00:00
[2025-01-28T09:35:12.409+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_data_ingestion', 'ingest_data', 'manual__2025-01-28T09:35:10.291418+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/dag_data_ingestion.py', '--cfg-path', '/tmp/tmptu9tzp6k']
[2025-01-28T09:35:12.411+0000] {standard_task_runner.py:105} INFO - Job 55: Subtask ingest_data
[2025-01-28T09:35:12.412+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1180) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-01-28T09:35:12.412+0000] {standard_task_runner.py:72} INFO - Started process 1181 to run task
[2025-01-28T09:35:12.446+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_data_ingestion.ingest_data manual__2025-01-28T09:35:10.291418+00:00 [running]> on host 7537dc2fd5ca
[2025-01-28T09:35:12.516+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_data_ingestion' AIRFLOW_CTX_TASK_ID='ingest_data' AIRFLOW_CTX_EXECUTION_DATE='2025-01-28T09:35:10.291418+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-28T09:35:10.291418+00:00'
[2025-01-28T09:35:12.517+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-28T09:35:12.528+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-28T09:35:12.528+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'pwd']
[2025-01-28T09:35:12.538+0000] {subprocess.py:99} INFO - Output:
[2025-01-28T09:35:12.540+0000] {subprocess.py:106} INFO - /tmp/***tmp29p2tmzp
[2025-01-28T09:35:12.540+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-28T09:35:12.561+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-28T09:35:12.562+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_data_ingestion, task_id=ingest_data, run_id=manual__2025-01-28T09:35:10.291418+00:00, execution_date=20250128T093510, start_date=20250128T093512, end_date=20250128T093512
[2025-01-28T09:35:12.586+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-28T09:35:12.599+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-28T09:35:12.600+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
